doc/BUILD

```diff
@@ -157,6 +158,7 @@ py_test_run_all_subdirectory(
     ],
     exclude = [
         "source/serve/doc_code/aws_neuron_core_inference_serve.py",
+        "source/serve/doc_code/aws_neuron_core_inference_serve_stable_diffusion.py"
```

Comment:
@ratnopamc just fyi, missing a comma at the end here

Reply:
@GeneDer thanks so much for catching that. Fixed and resubmitted.